deployment.go

```diff
@@ -28,7 +29,8 @@ type Deployment struct {
 	CurrentStep    int                 `json:"currentStep"`
 	TotalSteps     int                 `json:"totalSteps"`
 	AffectedApps   []string            `json:"affectedApps"`
-	Steps          [][]*DeploymentStep `json:"steps"`
+	Steps          [][]*DeploymentStep `json:"-"`
+	XXStepsRaw     json.RawMessage     `json:"steps"`
```

Comment:
Maybe the `XX` notation is conventional and I simply don't know about it, but I'd still love to see a comment next to this field saying that it is exported for internal purposes only.